base/deprecated.jl

```diff
@@ -1341,7 +1341,7 @@ end
 @deprecate xor(A::AbstractArray, B::AbstractArray)  xor.(A, B)
 
 # QuadGK moved to a package (#19741)
-function quadgk(args...)
+function quadgk(args...; kwargs...)
     error(string(quadgk, args, " has been moved to the package QuadGK.jl.\n",
```

Comment:
I know, I was surprised too. I copied that from another deprecation. I guess it's just because `args` is a tuple and calling `string` on a tuple gives it parens.